templates/coderd.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     app.kubernetes.io/component: {{ include "coder.serviceName" . }}
     app: {{ include "coder.serviceName" . }}
     coder.deployment: {{ include "coder.serviceName" . }}
-  annotations: {{ toYaml .Values.services.annotations | nindent 4 }}
+  annotations: {{ toYaml (merge (.Values.coderd.annotations | default (dict)) (.Values.services.annotations | default (dict))) | nindent 8 }}
```

Comment:
nindent 8 looks suspicious, please double check your indents

Reply:
I'm going to give `nindent 4` a shot. That was the value prior.

Reply:
looks like that didn't work. do we need to include `nindent` after each annotation being merged? e.g.

```yaml
annotations: {{ toYaml (merge (.Values.postgres.default.annotations | default (dict)) | nindent 4 (.Values.services.annotations | default (dict))) | nindent 4 }}
```

I feel like this is needed since each annotation we be on it's own new line.